src/index.js

```diff
@@ -8,7 +8,7 @@ import { Circle } from './modules/circle.js'
 let canvas = document.getElementById("game");
 let ctx = canvas.getContext("2d");
 let inputHandler = new InputHandler();
-let player = new Player(0, "player", 40, 40, 20, 1, "red");
+let player = new Player(0, "player", 40, 40, 20, 1, "green");
```

Comment:
Sadly I don't have per-map speeds or colors yet. Can we revert this for now? (ie - no changes in this file)